src/bench/bench.h

```diff
@@ -65,6 +65,7 @@ namespace benchmark {
             minCycles = std::numeric_limits<uint64_t>::max();
```

Comment:
Yeah let's change it to
```c++
State(std::string _name, duration _maxElapsed) :
name(_name),
maxElapsed(_maxElapsed),
count(0),
minTime(duration::max()), 
maxTime(duration::zero()),
minCycles(std::numeric_limits<uint64_t>::max()),
maxCycles(maxCycles = std::numeric_limits<uint64_t>::min()),
countMask(1),
lastCycles(0)
```
Makes it easier to see that everything is initialized.